modules/ROOT/pages/integrations/apollo-federation.adoc

```diff
@@ -44,38 +44,39 @@ const { url } = await startStandaloneServer(server);
 console.log(`🚀  Server ready at ${url}`);
 ----
 
-== Create a new project and install the Apollo Server
+== Setup
 
-*Create the project directory*
-
-Create a directory for a new project and `cd` into it:
+To proceed with the convertion, follow these steps:
 
+. Create a directory for a new project and `cd` into it:
++
 [source, bash]
 ----
 mkdir neo4j-graphql-subgraph-example
 cd neo4j-graphql-subgraph-example
 ----
 
-*Initialize the project*
-
-Initialize a new Node.js project with `npm`:
-
+. Initialize a new Node.js project with `npm`:
++
 [source, bash]
 ----
 npm init --yes
 npm pkg set type="module"
 ----
++
+[NOTE]
+====
+This how-to guide sets up the project using ES Modules, which allows the usage of features such as top-level `await`.
+====
 
-NOTE: This how-to guide sets up the project using ES Modules, which allows the usage of features such as top-level `await`.
-
-*Language specific setup*
-
-Follow the instructions below to set up with either TypeScript or JavaScript:
+. Choose TypeScript or JavaScript to proceed with the setup:
++
+[.tabbed-example]
```

Comment:
For some reason this is not rendering as tabs in the preview (actually, this whole page is rendering a bit funky)!

Reply:
fixed it now!